pkg/kubelet/container/runtime.go

```diff
@@ -231,12 +231,11 @@ func (id DockerID) ContainerID() ContainerID {
 type ContainerState string
 
 const (
+	ContainerStateCreated ContainerState = "created"
```

Comment:
Please update  PLEG to map this state to unknown for now so that 1) it doesn't panic, and 2) it doesn't generate an extra "container created" event https://github.com/kubernetes/kubernetes/blob/ef0c9f0c5b8efbba948a0be2c98d9d2e32e0b68c/pkg/kubelet/pleg/generic.go#L86